Cargo.toml

```
@@ -9,7 +9,7 @@ repository = "https://github.com/Ralith/hecs"
 readme = "README.md"
 keywords = ["ecs", "entity"]
 categories = ["data-structures", "game-engines", "no-std"]
-rust-version = "1.57"
+rust-version = "1.66"
```

```suggestion
rust-version = "1.65"
```

GAT will be stabilized even sooner !

---

Good catch, thanks!